src/test/java/com/softserveinc/ita/rozetka_test/OrderTest.java

```diff
@@ -4,21 +4,60 @@
 import com.softserveinc.ita.rozetka.page_objects.BasketPage;
 import com.softserveinc.ita.rozetka.page_objects.HomePage;
 import com.softserveinc.ita.rozetka.page_objects.OrderPage;
+import org.testng.Assert;
 import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
 
 public class OrderTest extends TestRunner {
 
-    //This added for future tests
     private OrderPage openOrderPage;
     private BasketPage openBasketPage;
 
-    //This added for future tests
     @BeforeMethod
     public void addProductToBasketOpenOrder() {
         openOrderPage = new HomePage()
                 .openHomePage()
                 .openProductByNumber(1)
                 .addProductToBasket()
                 .openOrderPage();
+
+        Assert.assertEquals(openOrderPage.getPageTitle(), "Оформлення замовлення", "Page header should be" +
+                " 'Оформлення замовлення'");
+    }
+
+    @Test
+    public void testEditOrder() {
+
+        openBasketPage = openOrderPage.startEditingProductsInBasket();
+
+        int orderProductCount;
+        orderProductCount = openBasketPage.getProductCount(1);
+        int orderProductSum;
+        orderProductSum = openBasketPage.getOrderProductSum(1);
+
+        int changedOrderProductCount;
+        int changedOrderProductSum;
+        openBasketPage.increaseProductCount(1);
+        changedOrderProductCount = openBasketPage.getProductCount(1);
+        changedOrderProductSum = openBasketPage.getOrderProductSum(1);
+
+        Assert.assertEquals(changedOrderProductCount, 2 * orderProductCount, "the new count for the product " +
+                "should be twice as much as the previous one");
```

Comment:
Incorrect product count

Reply:
I suggest leaving the available version as more informative